src/node.h

```diff
@@ -42,6 +42,12 @@
 
 #define NODE_DEPRECATED(msg, fn) V8_DEPRECATED(msg, fn)
 
+#include "imembase.h"
+
+#define malloc(size) ikmem_malloc((size_t)size)
+#define realloc(ptr, size) ikmem_realloc((void *)ptr, (size_t)size)
+#define free(ptr) ikmem_free((void *)ptr)
```

Comment:
you're right. it's not safe. just for test. for comparing with jemalloc and tcmalloc only.